app/views/layouts/snowcrash.html.erb

```diff
@@ -11,6 +11,9 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=0, minimum-scale=1.0, maximum-scale=1.0">
   <meta name="apple-mobile-web-app-capable" content="yes">
   <meta name="apple-mobile-web-app-status-bar-style" content="black">
+  <% if current_user %>
```

Comment:
Is this really needed? Can you ever be in SnowCrash layout w/o signing in first?